CRM/Contact/BAO/GroupContactCache.php

```diff
@@ -217,7 +217,7 @@ public static function loadAll($groupIDs = NULL, $limit = 0) {
    */
   public static function add($groupID) {
     // first delete the current cache
-    self::remove($groupID);
+    self::clearGroupContactCache($groupID);
```

Comment:
This is causing the test failure, because $groupID is (incorrectly) an array here.  `remove()` said it accepted an int as the first argument, but could also accept an array.  To fix this, I think you need to iterate through the array at line 208 so that `add()` is only passed an int.